crates/polars-lazy/src/physical_plan/expressions/gather.rs

```diff
@@ -101,7 +101,7 @@ impl PhysicalExpr for TakeExpr {
                             .collect_trusted()
                     },
                 };
-                let taken = ac.flat_naive().take(&idx)?;
+                let taken = ac.flat_naive().gather(&idx)?;
```

Comment:
Should the variable `taken` be renamed as well?
```suggestion
                let gathered = ac.flat_naive().gather(&idx)?;
```